README.md

```diff
@@ -0,0 +1,30 @@
+First Project
+
+This is my First Project at Microverse.
+

```

Comment:
- [ ] Kindly make sure you update your project tile to a meaningful related to the project you did to make your documentation professional.

- [ ] I also think it would be better if you update your project description to a meaningful one that can help users know what is about your project and this will make documentation more professional.